gallery/src/pageConfig.ts

```diff
@@ -43,12 +42,6 @@ import NxTablePage from './components/NxTable/NxTablePage';
 import NxTabsPage from './components/NxTabs/NxTabsPage';
 import NxStatefulTabsPage from './components/NxStatefulTabs/NxStatefulTabsPage';
 import NxTreeViewPage from './components/NxTreeView/NxTreeViewPage';
-import NxTreeViewRadioSelectPage from './components/NxTreeViewRadioSelect/NxTreeViewRadioSelectPage';
```

Comment:
Why are these gone?

Reply:
> Why are these gone?

They shouldn't be gone, that was a branch hygiene error on my part. Restored here:

https://github.com/sonatype/sonatype-react-shared-components/pull/596/commits/86a8a33a73fb8c4b822b97b3f2041f48a9e5ac0f